.circleci/config.yml

```diff
@@ -377,13 +377,28 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - unit_tests
-      - cdlang_tests
-      - coverage
-      - cpp-format-check
-      - cpp-lint-check
-      - license-check
-      - bazel-style-check
+      - publish-docker-build
```

Comment:
Unrelated to this issue; but we should consider to always build the image before running other jobs. This will catch problems like these early on. Main blocker for this is the tag, as we don't want to overwrite the main `:build` tag from PRs.